n = (gets || "").chomp.to_i

raise Exception.new("runtime error")